docs/setup/dbt-setup.md

```diff
@@ -1,19 +1,24 @@
-# dbt Project Setup
+# dbt project setup
 
 To set up a new project on dbt Cloud follow these steps:
 
 1. Give your new project a name.
 1. Click *Advanced settings* and in the *Project subdirectory* field, enter "transform"
 1. Select a data warehouse connection. (e.g. Snowflake, BigQuery, Redshift)
-1. For the *Development credentials* section you'll want to:
-    1. Under *Auth method* select *Key pair*
-    1. Enter your data warehouse username
-    1. Enter the private key amd private key passphrase
-    1. For more guidance, read [dbt's docs on connecting to Snowflake via key pair](https://docs.getdbt.com/docs/cloud/connect-data-platform/connect-snowflake#key-pair)
+1. For the *Development credentials* section you'll want to choose btween Snowflake OAuth or Key pair:
```

Comment:
```suggestion
1. For the *Development credentials* section you'll want to choose between Snowflake OAuth or Key pair:
```

It might be worth talking a bit about _why_ one would choose one or the other. In general, Snowflake OAuth is preferred for human users (which is what the development environment is for). That said, it's an enterprise dbt Cloud feature, so if you have a standard account, you'll need to use key pair.

Reply:
I agree with this, I'll add that in. Thank you Ian!